packages/components/src/components/drawer/drawer.jsx

```diff
@@ -45,7 +45,8 @@ class Drawer extends React.PureComponent {
     };
 
     render() {
-        const { children, className, footer, header } = this.props;
+        const { children, className, footer, header, footerHeight, headerHeight, contentClassName } = this.props;
+        const contentHeight = `calc(100% - ${headerHeight} - ${footerHeight})`;
```

Comment:
this is not used anywhere anymore so can be deleted I think? + we can remove the height props too now

Reply:
ah right.